orttraining/orttraining/python/ort_trainer.py

```diff
@@ -660,16 +663,38 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
         self._use_deterministic_compute = _use_deterministic_compute
         self.use_invertible_layernorm_grad = use_invertible_layernorm_grad
 
+        self._debug_conversion = _debug_conversion
+
         # use this special string to workaround a corner case that external loss_scale is passed into train_step as kwargs.
         # see prepare_input_and_fetches for more details.
         self.loss_scale_input_name = 'default_loss_scale_input_name'
 
         self._init_session()
 
+
+    def _debug_init_session(self):
+        self.no_postprocess_onnx_model = None
+        if not self._enable_internal_postprocess or not self._extra_postprocess:
+            self.no_postprocess_onnx_model = self.onnx_model_
+                
+            self.no_postprocess_session, self.no_postprocess_train_io_binding, self.no_postprocess_eval_io_binding, self.no_postprocess_output_name, _, self.no_postprocess_output_types = \
+                create_ort_training_session_with_optimizer(
+                    self.onnx_model_, self.device_,
+                    self.training_optimizer_name_, self.learning_rate_description_.name_, self.map_optimizer_attributes_,
+                    self.world_rank, self.world_size,
+                    self.gradient_accumulation_steps, bind_parameters=False,
+                    use_mixed_precision=self.use_mixed_precision, allreduce_post_accumulation=self.allreduce_post_accumulation_,
+                    deepspeed_zero_stage=self.deepspeed_zero_stage_,
+                    enable_grad_norm_clip=self.enable_grad_norm_clip_,
+                    frozen_weights=self.frozen_weights_, opset_version=self.opset_version_,
+                use_deterministic_compute=self._use_deterministic_compute)
+
     def _init_session(self):
         if self.onnx_model_ is None:
             return
 
+        if self._debug_conversion: self._debug_init_session()
```

Comment:
add a new line between the `if` and `then` clauses to mimic the same code style in this file